scripts/build-with-ldflags.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env sh
```

Comment:
`#!/usr/bin/env sh` I assume? og shell is hard to use/read

Reply:
nothing in here uses bash-specific stuff, doesn't really seem necessary?

Reply:
I'd lean towards having the same interpreter for all .sh files just for consistency. preferably bash

Reply:
ah, yea, we're aaaaalmost consistent on `#!/bin/bash` by the look of things.  might as well continue that trend I guess.